src/managers/TileManager.ts

```diff
@@ -38,16 +40,73 @@ class TileManager {
                 ),
                 session.seed,
             );
-            this.list.forEach((tile: Tile) => {
-                const state: TileState = TileState.Unused; // TODO
+
+            for (let i = 0; i < this.list.length; i++) {
+                const tile: Tile = this.list[i];
```

Comment:
don't know if it applies to loops, but shouldn't you always assign the type? (let i: int=0;...)

Reply:
Very good catch @Croxsy! You right there should be a type def.